tx.go

```diff
@@ -228,6 +259,15 @@ func UnmarshalTransaction(r io.Reader) (t Transaction, err error) {
 		}
 	}
 
+	t.ParentSeeds = make([]TransactionSeed, len(t.ParentIDs))
```

Comment:
Please update the web documentation to take into account the inclusion of parent seeds within each and every transaction.